public/app/features/expressions/components/SqlExpr.tsx

```diff
@@ -55,6 +55,12 @@ export const SqlExpr = ({ onChange, refIds, query }: Props) => {
     return () => resizeObserver.disconnect();
   }, []);
 
+  useEffect(() => {
```

Comment:
I'm not sure this is working right, look like when I load existing dashboards for which I had SQL queries, the query is being over written by the default text.

Reply:
To be clear I see this in dashboards.

Reply:
Good catch. I think this commit is orthogonal to the changes in this PR. I've cherry-picked it onto a separate branch and PR, here:  https://github.com/grafana/grafana/pull/103087

Given that, I was hoping to remove it from this branch completely. Was planning a rebase to do that @kylebrandt, but I could also add a revert commit at the end of the existing commits. WDYT?

Reply:
Thanks, good call moving to another PR. Would prefer revert